core/src/test/java/com/linecorp/armeria/common/util/BlockingTaskExecutorBuilderTest.java

```diff
@@ -0,0 +1,40 @@
+package com.linecorp.armeria.common.util;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.jupiter.api.Test;
+
+import com.linecorp.armeria.common.Flags;
+
+class BlockingTaskExecutorBuilderTest {
+
+    @Test
+    void testDefault() {
+        final ScheduledThreadPoolExecutor pool =
+                (ScheduledThreadPoolExecutor) BlockingTaskExecutor.builder().build().unwrap();
+        assertThat(pool.allowsCoreThreadTimeOut()).isTrue();
+        assertThat(pool.getKeepAliveTime(TimeUnit.MILLISECONDS)).isEqualTo(60 * 1000);
+        assertThat(pool.getCorePoolSize()).isEqualTo(Flags.numCommonBlockingTaskThreads());
+    }
+
+    @Test
+    void testOverride() {
+        final long keepAliveTime = 0;
+        final int numThreads = Flags.numCommonBlockingTaskThreads();
```

Comment:
We could use other value such as 42 so we actually override the default.

Reply:
Oh, I made a mistake. I'll fix it. Thank you!